great_expectations/core/util.py

```diff
@@ -69,23 +69,37 @@
 
 
 def nested_update(
-    d: Union[Iterable, dict], u: Union[Iterable, dict], dedup: bool = False
+    d: Union[Iterable, dict],
+    u: Union[Iterable, dict],
+    dedup: bool = False,
+    concat_lists: bool = True,
 ):
-    """update d with items from u, recursively and joining elements"""
+    """
+    Epdate d with items from u, recursively and joining elements. By default, list values are
+    concatenated without de-duplication. If concat_lists is set to False, lists in new dict
+    will replace those in new dict.
```

Comment:
Could we update the following:
* Typo in first word
* `new_dict` and `new_dict` in last sentence